2024-11-06 02:49:17,072 - ERROR - Error processing news item: 'str' object has no attribute 'get'
2024-11-06 02:49:17,072 - ERROR - Error processing news item: 'str' object has no attribute 'get'
2024-11-06 02:49:17,072 - ERROR - Error processing news item: 'str' object has no attribute 'get'
2024-11-06 02:49:17,072 - ERROR - Error processing news item: 'str' object has no attribute 'get'
2024-11-06 03:03:26,002 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:26,003 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:27,134 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:27,135 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:27,135 - INFO - Successfully saved study content for demo_1_20241106
2024-11-06 03:03:27,135 - INFO - Successfully processed news demo_1_20241106
2024-11-06 03:03:27,545 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:27,545 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:28,650 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:28,651 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:03:28,652 - INFO - Successfully saved study content for demo_2_20241106
2024-11-06 03:03:28,652 - INFO - Successfully processed news demo_2_20241106
2024-11-06 03:07:20,534 - INFO - Processing news item: demo_1_20241106
2024-11-06 03:07:20,641 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,642 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,726 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,726 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,727 - INFO - Successfully saved study content for demo_1_20241106
2024-11-06 03:07:20,727 - INFO - Successfully processed news demo_1_20241106
2024-11-06 03:07:20,727 - INFO - Processing news item: demo_2_20241106
2024-11-06 03:07:20,823 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,824 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,902 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,902 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:07:20,903 - INFO - Successfully saved study content for demo_2_20241106
2024-11-06 03:07:20,903 - INFO - Successfully processed news demo_2_20241106
2024-11-06 03:08:03,866 - INFO - Processing news item: demo_1_20241106
2024-11-06 03:08:04,037 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,037 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,130 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,131 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,131 - INFO - Successfully saved study content for demo_1_20241106
2024-11-06 03:08:04,131 - INFO - Successfully processed news demo_1_20241106
2024-11-06 03:08:04,131 - INFO - Processing news item: demo_2_20241106
2024-11-06 03:08:04,233 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,234 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,309 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,310 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:08:04,310 - INFO - Successfully saved study content for demo_2_20241106
2024-11-06 03:08:04,310 - INFO - Successfully processed news demo_2_20241106
2024-11-06 03:10:51,831 - INFO - Processing news item: demo_1_20241106
2024-11-06 03:10:51,947 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:51,948 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,052 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,053 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,053 - INFO - Successfully saved study content for demo_1_20241106
2024-11-06 03:10:52,053 - INFO - Successfully processed news demo_1_20241106
2024-11-06 03:10:52,053 - INFO - Processing news item: demo_2_20241106
2024-11-06 03:10:52,134 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,135 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,213 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,213 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:10:52,213 - INFO - Successfully saved study content for demo_2_20241106
2024-11-06 03:10:52,214 - INFO - Successfully processed news demo_2_20241106
2024-11-06 03:11:32,277 - INFO - Processing news item: demo_1_20241106
2024-11-06 03:11:32,532 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:32,532 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:33,667 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:33,668 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:33,669 - INFO - Successfully saved study content for demo_1_20241106
2024-11-06 03:11:33,669 - INFO - Successfully processed news demo_1_20241106
2024-11-06 03:11:33,669 - INFO - Processing news item: demo_2_20241106
2024-11-06 03:11:34,834 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:34,835 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:35,962 - ERROR - Vocabulary generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:35,962 - ERROR - Question generation error: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - '/home/runner/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/share/nltk_data'
    - '/opt/hostedtoolcache/Python/3.12.7/x64/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
**********************************************************************

2024-11-06 03:11:35,964 - INFO - Successfully saved study content for demo_2_20241106
2024-11-06 03:11:35,964 - INFO - Successfully processed news demo_2_20241106
